### ISSUE TITLE
Increase read-write space under live macOS

### ISSUE THREAD

**Issue by 2Shirt** [2Shirt/WizardKit#157]
```
------ Start Log -------

[2020-02-18 0218-0800 INFO] Initializing...
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk0s1
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk0s2
[2020-02-18 0218-0800 WARNING] Using SAT for smartctl for /dev/disk1
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk1s1
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk1s2
[2020-02-18 0218-0800 WARNING] Using SAT for smartctl for /dev/disk2
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk2s1
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk2s2
[2020-02-18 0218-0800 ERROR] Failed to get partition offset for /dev/disk2s3
[2020-02-18 0218-0800 WARNING] Disk attributes failure detected during diagnostics
[2020-02-18 0218-0800 INFO] Connecting to osTicket...
[2020-02-18 0218-0800 INFO]  
[2020-02-18 0218-0800 INFO]  
[2020-02-18 0218-0800 INFO] Is this correct? [Y/N]: Yes
[2020-02-18 0218-0800 INFO] CPU Test (Prime95)
[2020-02-18 0218-0800 CRITICAL] Major exception encountered
OSError: [Errno 28] No space left on device

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/hw-diags.py", line 10, in <module>
    wk.hw.diags.main()
  File "/usr/local/bin/wk/hw/diags.py", line 1223, in main
    run_diags(state, menu, quick_mode=False)
  File "/usr/local/bin/wk/hw/diags.py", line 1550, in run_diags
    function(state, *args)
  File "/usr/local/bin/wk/hw/diags.py", line 718, in cpu_mprime_test
    state.update_progress_pane()
  File "/usr/local/bin/wk/hw/diags.py", line 400, in update_progress_pane
    _f.write('\n'.join(report))
OSError: [Errno 28] No space left on device
[2020-02-18 0218-0800 WARNING] Please let support know by opening an issue on Gogs

------- End Log --------
--- Start debug info ---

[System]
  FQDN                     macbook-pro.local
  Architecture             ('64bit', '')
  Machine                  x86_64
  Platform                 Darwin-15.5.0-x86_64-64bit
  Python version           3.7.6
  Python sys.argv          ['/usr/local/bin/hw-diags.py']

[Environment Variables]
  CI_DEFAULT_OPENCL_USAGE  0
  COLORFGBG                7;0
  COLORTERM                truecolor
  DYLD_NO_FIX_PREBINDING   1
  ITERM_PROFILE            Hardware Diagnostics
  ITERM_SESSION_ID         w0t0p0:3C258690-0675-4409-B203-EAEC4147C42B
  LANG                     en_US.UTF-8
  LC_ALL                   en_US.UTF-8
  OS_INSTALL               1
  PATH                     /usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/local/opt/util-linux/sbin:/usr/local/opt/util-linux/bin:/usr/local/opt/ruby/sbin:/usr/local/opt/ruby/bin:/usr/local/opt/e2fsprogs/sbin:/usr/local/opt/e2fsprogs/bin:/usr/bin:/bin:/usr/sbin:/sbin
  PWD                      /usr/local/bin
  SHELL                    /bin/sh
  SHLVL                    4
  TERM                     screen
  TERM_PROGRAM             iTerm.app
  TERM_PROGRAM_VERSION     3.1.7
  TERM_SESSION_ID          w0t0p0:3C258690-0675-4409-B203-EAEC4147C42B
  TMUX                     /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.z9ZumaYC.socket,1153,0
  TMUX_PANE                %0
  XPC_FLAGS                0x0
  XPC_SERVICE_NAME         0
  _                        /usr/local/bin/hw-diags.py
  __OSINSTALL_ENVIRONMENT  1
  __PYVENV_LAUNCHER__      /usr/local/bin/python3

---- End debug info ----
```

**Comment by 2Shirt** [comment]
Create RAMDisk: `hdiutil attach -nomount ram://size` (returns disk path e.g. `/dev/disk2`)
Where size = MiB * ( (1024^2) / 512) or MiB*2048

Format RAMDisk: `diskutil eraseVolume HFS+ 'RAM Disk' path-to-disk`

Combined example (128MiB): `diskutil eraseVolume HFS+ 'RAM Disk' $(hdiutil attach -nomount ram://262144)`

**Comment by 2Shirt** [comment]
Addressed in the nebulous commit: 5b4139e3c807ac3ac2f56b2bbd5590f4e1a04c79
